src/Surfnet/StepupMiddleware/ManagementBundle/Tests/Validator/Fixtures/invalid_reconfigure_institution_request/missing_allowed_second_factors_option.php

```diff
@@ -23,6 +23,7 @@
         'surfnet.nl' => [
             'use_ra_locations' => true,
             'show_raa_contact_information' => true,
+            'verify_email' => false,
```

Comment:
Why did you set verify_email to false by default in the `invalid_reconfigure_institution_request` fixtures?

Reply:
It's just an arbitrary value.